README.md

```diff
@@ -28,30 +28,30 @@ If you need to use an illustration that exceeds this value, you can compress it
 2. Copy the file you want to make available for download to the same directory as the article where it will be inserted. For instance, to attach a spreadsheet within an article located at `docs/pt/tutorials/Payments/Payment Settings/list-of-payment-providers-by-country.md`, you need to save it in the directory `docs/pt/tutorials/Payments/Payment Settings.`
 3. Go to the [branches page](https://github.com/vtexdocs/help-center-content/branches), and under **Your branches** section, click on the name of the branch you created.
 
-![Branches_page](https://raw.githubusercontent.com/vtexdocs/help-center-content/main/readme-download-illustration-1.png)
+![Branches_page](https://raw.githubusercontent.com/vtexdocs/help-center-content/main/readme-images/readme-download-illustration-1.png)
 
 4. Locate the folder where you saved the file and click on it.
 
-![File_folder](https://raw.githubusercontent.com/vtexdocs/help-center-content/main/readme-download-illustration-2.png)
+![File_folder](https://raw.githubusercontent.com/vtexdocs/help-center-content/main/readme-images/readme-download-illustration-2.png)
 
 5. Right-click on the **Raw** option and select **Copy link address (Copiar endereço do link)**.
 
-![Raw_link_information](https://raw.githubusercontent.com/vtexdocs/help-center-content/main/readme-download-illustration-3.png)
+![Raw_link_information](https://raw.githubusercontent.com/vtexdocs/help-center-content/main/readme-images/readme-download-illustration-3.png)
 
-> ℹ️ Address link example: `https://github.com/vtexdocs/help-center-content/raw/docs/pt/tutorials/Payments/Payment%20Settings/Payment%20Provider%20x%20Countries%20-%20August%202024.xlsx`
+> ℹ️ Address link example: `https://github.com/vtexdocs/help-center-content/raw/{branch-name}/docs/pt/tutorials/Payments/Payment%20Settings/Payment%20Provider%20x%20Countries%20-%20August%202024.xlsx`
```

Comment:
```suggestion
> ℹ️ Address link example: `https://github.com/vtexdocs/help-center-content/raw/main/docs/pt/tutorials/Payments/Payment%20Settings/Payment%20Provider%20x%20Countries%20-%20August%202024.xlsx`
```

Reply:
Tha branch name has to be `main` because the content is going to end up in the main anyway.